src/index.html

```diff
@@ -12,6 +12,39 @@
 </head>
 
 <body>
+  <materials-dialog-controller></materials-dialog-controller>
+  <button id='btn'>Test</button>
 </body>
 
+<script>
```

Comment:
This was for testing but, needs to be put in materials-site

Reply:
i do it